roles/hostedbm/tasks/validations.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Ensure hostedbm_cluster_name is defined
```

Comment:
From hear till line 24 you can combine it to one task with loop and dictionary
{condition evaluation [true/false],failed messages}

Reply:
thank you for the great suggestion, done.